content/en/tools/digital-currencies/transferring/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+date: '2023-06-23'
+lastmod: '2023-06-23'
+section_menu: tools
+menu:
+  tools:
+    name: "Working with Token Transfers"
+    weight: 1200
+    parent: digital-currencies
+    identifier: digital-currencies-tokens-transferring
+    description: "Digital Currencies documentation describing how to transfer tokens via the GUI"
+title: "Working with Token Transfers"
+---
+
```

Comment:
maybe adding {{ childpages }} ?

Reply:
In this case I didn't, as then it would add Approving or Rejecting Transfer Requests as well under the "The following transfer methods are available:" line.